src/reducers/org.js

```diff
@@ -994,11 +1057,26 @@ export default (state = new Map(), action) => {
       return createLogEntryStart(state, action);
     case 'UPDATE_LOG_ENTRY_TIME':
       return updateLogEntryTime(state, action);
+    case 'SET_SEARCH_FILTER_INFORMATION':
+      return setSearchFilterInformation(state, action);
+    case 'SET_SEARCH_ALL_HEADERS_FLAG':
+      return setSearchAllHeadersFlag(state, action);
+
     default:
       return state;
   }
 };
 
+function getTodoKeywordSetsAsFlattenedArray(state) {
```

Comment:
This function sounds more like a helper function for org_utils.js...

Reply:
True. I'll move it there.

Reply:
Thanks!